CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Pipeline has been re-implemented in [Nextflow DSL2](https://www.nextflow.io/docs
 - [#76](https://github.com/nf-core/metapep/pull/76) - Restructure and rename `CHECK_INPUT` subworkflow to `PREPROCESS_INPUT` and move some channel logic into the subworkflow. Also `CHECK_SAMPLESHEET` module is renamed to `INPUT_TO_DATAMODEL` to have a more descriptive name and circumvent clashing with the nf-core template.
 - [#79](https://github.com/nf-core/metapep/pull/79) - Replaced multi-package BioContainers by single-package pandas BioContainer and updated containers with old pandas versions
 - [#82](https://github.com/nf-core/metapep/pull/82) - Add label `error_retry` to process `DOWNLOAD_PROTEINS`
+- [#85](https://github.com/nf-core/metapep/pull/85) - Changed the output of `UNIFY_MODEL_LENGTHS` to a logfile instead of a tsv
```

Comment:
I would tend in such cases to add descriptions more focussing on what was added in comparison to before, since the change is only with respect to some of the most recent changes and would be difficult to understand without knowing the individual PRs. Just a thougth since it was anyway on the list to clean up the CHANGELOG

Reply:
Thats true! I will keep it in mind when concatenating the changes to a more meaningful changelog :)